e2e/resources/old-app-crd.yaml

```diff
@@ -0,0 +1,241 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Would you mind renaming this file to app-crd-v0.8.0 or something like that ?

Reply:
Thanks. Done